vtysh/extract.pl.in

```diff
@@ -98,7 +98,10 @@ sub scan_file {
         elsif ($file =~ /lib\/if\.c$/) {
             $protocol = "VTYSH_INTERFACE";
         }
-        elsif ($file =~ /lib\/(filter|filter_cli|lib_vty)\.c$/) {
+        elsif ($file =~ /lib\/(filter|filter_cli)\.c$/) {
+            $protocol = "VTYSH_RMAP";
```

Comment:
ldpd uses ACLs but not route-maps :(

I think you'll need to add a new group of daemons like `VTYSH_FILTER`.

Reply:
@rwestphal thanks, I refactored the commits and shortened the daemon list to send commands to one by one.

Also I alphabetically ordered the list so it is easier to spot the missing entry.